src/installation/install_epinio_manual.md

```diff
@@ -0,0 +1,120 @@
+# Install Epinio (the manual way)
+
+As described [in the top level document](src/installation/installation.md#installation-methods), this method is intended for production level setups where operators want to have full control of the process.
+
+## Prerequisites
+
+Epinio needs a number of external components to be running on your cluster in order to work. You may already have those deployed, otherwise follow the instructions here to deploy them.
+
+Important: Some of the namespaces of the components are hardcoded in the Epinio
+code and thus are important to be the same as described here. In the future this
+may be configurable on the Epinio Helm chart.
+
+### Linkerd
+
+- Optional
+
+Download the linkerd cli from here: https://github.com/linkerd/linkerd2/releases/tag/stable-2.10.2
```

Comment:
Do we want to have specific versions specified here or better describe at least `linkerd >= 2.10`  or tested with ...?

Reply:
Hm, I don't know. It's hard to provide links this way. Ideally we should document the minimum version you are right. But no easy links in that case.

Reply:
I split the link with the version explaining `2.10.2` is known to work.